PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/NativeImageAsset.java

```diff
@@ -20,6 +20,12 @@ public NativeImageAsset(int w, int h, int minWidth, int minHeight) {
         hmin = minHeight;
     }
 
+    public NativeImageAsset(int minWidth, int minHeight) {
+        super(REQUEST_ASSET.IMAGE);
+        wmin = minWidth;
```

Comment:
Shouldn't this one set `v` and `h` to be `-1` for instance, to make sure they are skipped in the json serialization?

Reply:
`w` and `h` are already initialized as `-1`. You can find them lower in this class.

Reply:
Oh, great 👍